src/pages/index.astro

```diff
@@ -9,6 +9,8 @@ import CallToAction from 'components/index/CallToAction.astro';
 ---
 
 <Layout title="DELTA">
+  <RegisterForm client:only="solid-js"/>
```

Comment:
Please render it on separate page and add button in header that will lead to registration.